messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryUpdateEmitter.java

```diff
@@ -301,6 +312,12 @@ public Builder updateMessageMonitor(
             return this;
         }
 
+        public Builder spanFactory(@Nonnull SpanFactory spanFactory) {
```

Comment:
Furthermore, have you been consequent with adding the `@Nonnull` annotation to all builder methods you've added?